src/main/java/frc/robot/constants/JoystickConstants.java

```diff
@@ -25,6 +26,9 @@ private JoystickConstants() {
     public static final double DRIVE_REG_Y_DEADZONE = 0.10;
     public static final double DRIVE_REG_ROT_DEADZONE = 0.15;
 
+    //Maximum value the joysticks can output, need to be calculated. TODO
```

Comment:
Create an issue for this and paste the link in the comment.

Reply:
We realized that we don't need to calculate this, so we removed the comment.